docs/cndb_omf.md

```diff
@@ -247,13 +283,15 @@ Part 4: CNDB compaction (txm records are skippped for brevity)
 
 ### Incremental spills
 
-Incremental spill refers to the fact that a spill constructs and plugs in all the kvsets of a child node before proceeding to
-the next child node. A full spill operation can take a while, but it doesn't have to hold up all split/join/kcompact/kvcompact
-operations until it finishes. Only the child node to which the spill is writing data needs to be locked from participating in
-any maintenance operations.
+Incremental spill refers to the fact that a spill constructs and plugs in all the kvsets of a child
+node before proceeding to the next child node. A full spill operation can take a while, but it
+doesn't have to hold up all split/join/kcompact/kvcompact operations until it finishes. Only the
+child node to which the spill is writing data needs to be locked from participating in any
+maintenance operations.
```

Comment:
Not really sure what "plugs in all the kvsets" means...

s/locked/prevented/

Reply:
I meant create kvsets and add them to the node list (i.e. update the cn tree).

Fixed it - made it explicit.